src/Console/Command/FixCommand.php

```diff
@@ -184,7 +184,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $progressOutput = $showProgress && $stdErr
-            ? new ProcessOutput($stdErr, $this->eventDispatcher)
+            ? new ProcessOutput($stdErr, $this->eventDispatcher, count($resolver->getFinder()))
```

Comment:
if it's not rewindable we could convert it to array anyway (even if it would be big memory-consuming array)
next argument to have 2 `ProcessOutput` implementations